tox.ini

```diff
@@ -24,6 +24,30 @@ setenv =
 # passed from Tox's invocation environment to the testing environment
 passenv = TOXENV RESOLWE_* RESOLWEBIO_* DOCKER_*
 
+[testenv:docs]
+commands =
+# install documentation requirements
+    pip install --process-dependency-links .[docs]
+# build documentation
+# NOTE: After https://github.com/sphinx-doc/sphinx/pull/2649 is accepted, use:
+# python setup.py build_sphinx --warning-is-error
+    sphinx-build -E -W docs/ build/sphinx/html
+
+[testenv:linters]
+# run all linters to see their output even if one of them fails
+ignore_errors = True
+setenv =
+    PYTHONPATH = {toxinidir}/resolwe_bio/tools
```

Comment:
It might be worth adding a comment why setting `PYTHONPATH` is necessary. Something like:
`# enable pylint to find modules in the tools package even though it is not installed`